lib/app/app.rb

```diff
@@ -24,5 +24,6 @@ module App
     worker.start!
   rescue App::PreflightCheck::CheckFailure => e
     Utility::Logger.error("Preflight check failed: #{e.message}")
+    exit(-1)
```

Comment:
the rule of thumb is that if anything bad happens when you exit you need something different than zero so the supervisor knows the app is unhealthy